src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala

```diff
@@ -350,13 +380,261 @@ class ConsignmentStatusServiceSpec extends AnyFlatSpec with MockitoSugar with Re
     val expectedStatusValue = "InvalidStatusValue"
 
     val updateConsignmentStatusInput =
-      ConsignmentStatusInput(expectedConsignmentId, expectedStatusType, expectedStatusValue)
+      ConsignmentStatusInput(expectedConsignmentId, expectedStatusType, Some(expectedStatusValue))
 
     val thrownException = intercept[Exception] {
       consignmentService.updateConsignmentStatus(updateConsignmentStatusInput).futureValue
     }
 
-    thrownException.getMessage should equal(s"Invalid ConsignmentStatus input: either '$expectedStatusType' or '$expectedStatusValue'")
+    thrownException.getMessage should equal(
+        s"Invalid ConsignmentStatus input: either '$expectedStatusType' or '$expectedStatusValue'"
+    )
+  }
+
+  "updateConsignmentStatus" should s"call fileStatus repo in order to generate a statusValue when only a consignmentId and 'Upload' statusType have been passed in" in {
```

Comment:
Maybe re-word test title slightly to: `... repo to generate a statusValue when a consignmentId and 'Upload' ...`